spec/rsteamshot/screenshot_spec.rb

```diff
@@ -82,4 +82,31 @@
       expect(json['comment_count']).to eq(comment_count)
     end
   end
+
+  describe '#file_size_in_bytes' do
+    it 'converts file_size string in B into integer of bytes' do
+      expect(screenshot).to receive(:file_size).and_return('123 B')
```

Comment:
I usually see `allow` instead of `expect` paired with `to receive`. I guess this works though!